### ISSUE TITLE
Auto skip intro/credits toggle

### ISSUE THREAD

**Issue by GillyTheGhillie** [jarnedemeulemeester/findroid#851]
**Describe the solution you'd like**
It would be nice to have an option in the settings to automatically skip intros and credits instead of having to manually press the skip intro/credits button. As I understand it the intro/credit skip in Findroid simply uses the timestamp data from jumoog's plugin rather than using the plugin to handle the actual skipping.

**Describe alternatives you've considered**
I believe the default auto skip in the plugin settings for jumoog's intro skipper currently only works with the web player and players based off of that (e.g. official android app web player)

**Comment by foux** [comment]
Don't forget that segments will be a core feature of jellyfin in 10.10. So perhaps we should stop looking specifically in Intro Skipper functionnality (and API), which seems a loss of time with 10.10 so close.

**Comment by UriSheinman** [comment]
I think this has a setting in the plugin...

**Comment by foux** [comment]
WIth Jellyfin 10.10 only a week away, it is important to stop thinking about the plugin, and start thinking with the internal Jellyfin implementation of the segments.

**Comment by UriSheinman** [comment]
> WIth Jellyfin 10.10 only a week away, it is important to stop thinking about the plugin, and start thinking with the internal Jellyfin implementation of the segments.

I think it'll take sometime until everyone transition, it's good to have a backup for legacy users instead of just throwing away the work done...

**Comment by GillyTheGhillie** [comment]
> I think this has a setting in the plugin...

My understanding (and this may be wrong) is that Findroid only uses the timestamp data from the plugin, it doesn't use any of its functionality. I've tried toggling auto skip on and off and it hasn't made any difference to the behaviour in Findroid.

**Comment by GillyTheGhillie** [comment]
> WIth Jellyfin 10.10 only a week away, it is important to stop thinking about the plugin, and start thinking with the internal Jellyfin implementation of the segments.

I still think auto skip could be something worth implementing for the official segment implementation.

**Comment by foux** [comment]
> > WIth Jellyfin 10.10 only a week away, it is important to stop thinking about the plugin, and start thinking with the internal Jellyfin implementation of the segments.
> 
> I still think auto skip could be something worth implementing for the official segment implementation.

I agree with that. I'm just saying there's no need to spend time on the plugin, when dozens or other plugins will be created with the implementation of segments.
